### DISCUSSION TITLE
Wonderful idea, needs some work though

### DISCUSSION THREAD

**Discussion by dominik-myszkowski** [LiveDuo/destack#115] [General]
Hey guys, what a wonderful project idea, I love it. It is something that is missing.
But... currently there are tons of errors which make it unusable... I tried both destack-starter and destack-starter-beta and I was unable to make any project in beta, and in the former only few themes/components worked without ruining the interface.
When these fatal errors are corrected it would be a wonderful tool, I am sure!

**Comment by LiveDuo** [comment]
@dominik-myszkowski If you are facing issues please open a Github issue with more details about the problem your are encountering. Include what errors you see and hopefully a way to reproduce to help further.
